apps/src/componentLibrary/common/styles/colors.css

```diff
@@ -0,0 +1,212 @@
+@import "primitiveColors.css";
+
+/* This file consists of Semantic and Utility colors,
+ if you need DSCO colors, you need to import and use this file.
+ */
+
+/* Colors are taken from DISCO Variables New Figma File - Semantic colors and Utility colors variables collections
+ via variables2css plugin that generates css code based on Figma variables values and names.
+ We're using css here in order to support not only scss, but also css files.
+ Also, stylelint fix might be required for autogenerated colors, which can be done in one command.
+ */
+
+/* Semantic Colors */
+
+/* Light theme Semantic colors 
+(Light theme is a default one, that's why here's also :root rule)
+*/
+:root, [data-theme="Light"] {
```

Comment:
Ah, this answers my question on https://github.com/code-dot-org/code-dot-org/pull/59607 🙂 - slick!